var/spack/repos/builtin/packages/xqilla/package.py

```diff
@@ -18,6 +18,11 @@ class Xqilla(AutotoolsPackage, SourceforgePackage):
     variant('debug', default=False, description='Build a debugging version.')
     variant('shared', default=True, description='Build shared libraries.')
 
+    # see https://sourceforge.net/p/xqilla/bugs/48/
+    patch('xerces-3-2-0.patch', when="@:2.3.3")
+    patch('xerces-3-2-0_1.patch', when="@:2.3.3")
+    patch('gcc11.patch', when="%gcc@11:")
```

Comment:
LGTM

To avoid pulling in the patch files, you could also do:
```python
patch('https://src.fedoraproject.org/rpms/xqilla/raw/1f2f53305f429aa3db2ab078d9613fbc367b402d/f/0004-xerces-3.2.0-casts.patch', sha256='78997e098f041bf41def6fab436ea406b2dceaa15ae3ec8a8d2aa7ed356a0bb9', when='@:2.3.3')
patch('https://src.fedoraproject.org/rpms/xqilla/raw/1f2f53305f429aa3db2ab078d9613fbc367b402d/f/0005-xqilla-gcc11.patch', sha256='52e5f03012fe9ae5b0f90d04eff042fb2082aa8f366a47d9e6be0d452de87b73', when='%gcc@11:')
```

Reply:
Thanks for taking a look! I didn't know fedora had those patches already, that's good to know for the future. I personally tend to avoid these kinds of network accesses - aren't remote patches mostly done when the licensing could pose an issue?

Reply:
We often use URL patches to avoid pulling them into the repository as they will be stored on Spack's source mirror anyway. It's up to you, though. Just let me know whether you want to make the change or want to stay with the file patches.

Reply:
That's a good point, if it can be cached, then it is indeed cleaner to add this by url. Have updated the PR now.